Makefile

```diff
@@ -11,9 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-DOCKER_TAG ?= 1.5.1
-SIDECAR_TAG ?= 1.5.1
-VERSION_TAG ?= 1.5-1
+export DOCKER_TAG ?= 1.6.0
+export SIDECAR_TAG ?= 1.6.0
+# Get version and release from DOCKER_TAG
+dot-delimiter = $(word $2,$(subst ., ,$1))
+export VERSION = $(call dot-delimiter, ${DOCKER_TAG}, 1).$(call dot-delimiter, ${DOCKER_TAG}, 2)
```

Comment:
Will this also handle DOCKER_TAG being set to something without the semantic release versions? Like "latest", "nightly"?

Reply:
Ah yeah then that wouldn't work. I'd have to rethink this through then. What would version and release be for nightly?

Reply:
You will probably need to keep the RPM file to be numbered with the default version (1.6-0 for e.g.). Only the docker images can be built and saved with the non versioned tags.

Reply:
I've updated so it checks if RELEASE is empty then it'll use default values. RELEASE would be empty if DOCKER_TAG doesn't have dot delimiters